src/dict/loaddictionaries.hh

```diff
@@ -39,7 +40,7 @@ public:
   /// Empty string means to exception occurred
   std::string const & getExceptionText() const
```

Comment:
This returns a ref to temporal object. 

Should be `std::string getExceptionText()` instead.

https://github.com/xiaoyifang/goldendict-ng/actions/runs/13459304050/job/37610490793#step:6:429